### PR TITLE
fix: conversation icons adjustments

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Fix federation icon size
* Move icon depicting a conversation type, from details slot to the user_status place
* Adjust conversation icon for invitations

### 🖼️ Screenshots / Screencasts

| 🏚️ Before | 🏡 After |
|------------|----------|
Screenshot from 2024-02-26 15-44-08 | image
Screenshot from 2024-02-26 15-43-50 | image
Screenshot from 2024-02-26 14-33-11 | Screenshot from 2024-02-26 14-32-40 Screenshot from 2024-02-26 14-37-52
image | image

### 🏁 Checklist

- [x] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
- [x] 🖌️ Design was reviewed, approved or inspired by the design team

### CODE REVIEW

File: src/components/ConversationIcon.vue [hunk 1]
```diff
@@ -167,10 +179,11 @@ export default {
 		iconClass() {
 			if (this.item.isDummyConversation) {
 				// Prevent a 404 when trying to load an avatar before the conversation data is actually loaded
-				// Also used in new conversation dialog
+				// Also used in new conversation / invitation handler dialog
+				const isFed = this.item.isFederatedConversation && 'icon-conversation-federation'
```
Comment (DorraJaouad) [inline src/components/ConversationIcon.vue:183]: Should we consider it a type from now on ?

File: src/components/ConversationIcon.vue [hunk 2]
```diff
@@ -212,6 +225,17 @@ export default {
 			return this.item.type === CONVERSATION.TYPE.ONE_TO_ONE
 		},
 
+		conversationType() {
+			if (this.item.remoteServer) {
+				return { icon: WebIcon, label: t('spreed', 'Federated conversation') }
+			} else if (this.item.type === CONVERSATION.TYPE.PUBLIC) {
+				return { icon: LinkVariantIcon, label: t('spreed', 'Public conversation') }
+			} else if (this.item.type === CONVERSATION.TYPE.GROUP) {
+				return { icon: AccountMultipleIcon, label: t('spreed', 'Group conversation') }
```
Comment (nickvergessen) [inline src/components/ConversationIcon.vue:234]: lets not show this

File: src/components/ConversationIcon.vue (hunk 1 above)
Comment (Antreesy) [reply src/components/ConversationIcon.vue:183]: Might complicate things and on the API side

**Review by DorraJaouad — APPROVED** [review]
Tested

**Review by jancborchardt — COMMENTED** [review]
Looks nice! :) Would of course be cool if it’s not a visible "border" but a cutout instead –though that’s not something we can do on our own, probably have to wait for some CSS magic. ^^

**Pull request merged**